### ISSUE TITLE
Sharp colors.

### ISSUE THREAD

**Issue by SinadoDev** [arXiv/html_feedback#5716]
### Description

The red barner, white text on black background mix, is quite painful to the eyes. Consider more soothing colors. Could you add the option to switch modes for different users with different accessibility requirements?

### (Optional:) Please add any files, screenshots, or other information here.

<img width="1909" height="611" alt="Image" src="https://github.com/user-attachments/assets/e5b6decf-21aa-4d1f-b58f-29eed0e80cd2" />

### (Required) What is this issue most closely related to? Select one.

Choose One

### Internal issue ID

f1c484b0-4e85-4b97-8b5a-2a9a21424055

### Paper URL

https://arxiv.org/html/2501.02303v1

### Browser

Chrome/141.0.0.0

### Device Type

Desktop

**Comment by dginev** [comment]
The page scaffold for HTML papers has been refreshed for new articles starting March 2026. 
The article reported in this issue (thank you!) has been updated to include that scaffold.

A full update of all arXiv HTML articles will follow at a later date.

You can use the new header buttons to hide/show the navigational menu, as well as the header itself. That choice will be saved for future visits in the same browser. The "Report Issue" button is now only available in the page header.

---

As to color themes, that may only be considered in the long-term future, but we'll track it separately.
